executor/src/constant_evaluator/mod.rs

```diff
@@ -176,11 +176,8 @@ mod test {
         let src = r#"
             let N = 8;
             namespace F(N);
-            pol constant LAST(i) { match i {
-                N - 1 => 1,
```

Comment:
We cannot match against values any more.

Reply:
No, this is just not how patterns work (I think). Rust allows constants, but only constants and not expressions containing constants.